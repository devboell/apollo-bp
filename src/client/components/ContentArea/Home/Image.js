import styled from 'styled-components'

const Image = styled.img`
  height: 700px;
`

export default Image
