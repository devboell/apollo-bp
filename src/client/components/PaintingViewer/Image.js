import styled from 'styled-components'

const Image = styled.img`
  max-width: 100%;
  max-height: 700px;
  margin: auto;
`

export default Image
