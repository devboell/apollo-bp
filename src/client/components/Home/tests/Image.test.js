import React from 'react'
import 'jest-styled-components'

import Image from '../Image'

it('Image, styled component snapshot', () => {
  const wrapper = shallow(<Image />)
  expect(wrapper).toMatchSnapshot()
})
