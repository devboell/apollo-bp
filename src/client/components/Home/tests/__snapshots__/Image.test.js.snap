// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image, styled component snapshot 1`] = `
.OThze {
  height: 700px;
}

<img
  className="sc-bdVaJa OThze"
/>
`;
